python/python/glide/async_commands/core.py

```diff
@@ -2326,6 +2326,33 @@ async def zscore(self, key: str, member: str) -> Optional[float]:
             await self._execute_command(RequestType.ZScore, [key, member]),
         )
 
+    async def zmscore(
+        self,
+        key: str,
+        members: List[str],
+    ) -> List[Optional[float]]:
+        """
+        Returns the scores associated with the specified `members` in the sorted set stored at `key`.
+
+        See https://valkey.io/commands/zmscore for more details.
+
+        Args:
+            key (str): The key of the sorted set.
+            members (List[str]): A list of members in the sorted set.
+
+        Returns:
+            List[Optional[float]]: A list of scores of the `members`.
```

Comment:
```suggestion
            List[Optional[float]]: A list of scores corresponding to `members`.
```
tell me what you think

Reply:
sounds good